models/university.py

```diff
@@ -1,11 +1,25 @@
-from sqlalchemy import (MetaData, Column, Table, Integer, VARCHAR, ForeignKey)
+from models import rector, faculty, hostel, requisites, user_request_review
 
+from sqlalchemy import (Column, INTEGER, VARCHAR, ForeignKey)
+from sqlalchemy.orm import relationship
 
-metadata_obj = MetaData()
+from db import Base
 
+class University(Base):
+    __tablename__ = "university"
 
-university = Table('university', metadata_obj,
-             Column('university_id', Integer, primary_key = True),
-             Column('university_name', VARCHAR(255)),
-             Column('logo', VARCHAR(255)),
-             Column('rector_id', Integer, ForeignKey("rector.rector_id")))
+    university_id = Column(INTEGER, primary_key=True)
+    university_name = Column(VARCHAR(length=255))
```

Comment:
university_name can't be with null value

Reply:
But the old declaration was `Column('university_name', VARCHAR(255))`. Isn't current declaration equivalent? O_o

Reply:
The old one was wrong. When we update or change smth, we need to pay attention on main.sql file